nautilus_trader/model/events/order.pyx

```diff
@@ -855,7 +856,7 @@ cdef class OrderDenied(OrderEvent):
             "client_order_id": obj.client_order_id.value,
             "reason": obj.reason,
             "event_id": obj.id.value,
-            "ts_event": obj.ts_init,
+            "ts_event": obj.ts_event,
```

Comment:
This is probably the right thing to do, but it'll be the same as `ts_init`, as there's no separate event timestamp for this one.